src/net.cpp

```diff
@@ -655,6 +655,11 @@ void CNode::SetAddrLocal(const CService& addrLocalIn) {
     }
 }
 
+std::string CNode::GetLogString() const
+{
+    return fLogIPs ? addr.ToString() : strprintf("%d", id);
```

Comment:
Please always log the id, no matter if logging IPs is requested or not. I use greps for `peer=xxx` all the time to find out all communication through a specific connection (and a peer might have connected multiple times, so addr is not enough)

Reply:
Not sure I understand... This function does not remove/break any `peer=xxx`, it's just used to replace IPs with IDs where it's possible.

Reply:
Ahh sorry I misunderstood the changes :)